docs/CHANGELOG.md

```diff
@@ -6,12 +6,128 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
+- Close panels when pressing <kbd>ESC</kbd>
+- Shift-click to spend all stat points
+- Shift-click to clear readied spell
+- Shift-click consumables to move them between the belt and inventory
+- Shift-click equipment equip/unequip them
+- Preserve hotkeys and active spell across games
+- Set default active spell for new heroes
+- Apply sound volume to videos
+- DIABDAT.MPQ may now be upper case
 - Improved gamepad support:
   - D-Pad / Left stick navigation now works in all menus and is more responsive.
   - START + SELECT now opens the main menu.
   - Right stick mouse emulation has been improved.
   - Gamepads plugged in while the game is running can be used.
   - All connected gamepads can be used (previously, only the first one).
+- Add `--ttf-dir` and `--ttf-name` to allow for specifying a different UI font
+
+### Optional improvements
+- Improved transparency
+- Experience bar
+- Enemy health bar
+- Auto equip all item types
+- Auto pickup gold
+- Run in town (also in multiplayer)
+- Disable friendly fire
+- Show monster type
+- Disable walking sounds
+- No quest randomization
+- Adria refills mana
+- Barbarian and Bard heroes in Diablo
+- Controller mapping
+- Configurable network port
+- Support for portable installs
+
+### Platforms
+- Switch Windows releases to 64bit
+- Added support for PlayStation Vita
+- Added support for Nintendo 3DS
+- Added support for DragonFly BSD
+- Added AppImage for Linux
+
+### Bugfixes
+- Incorrect graphics under cave doors in multiplayer
+- Catacombs doors never having traps
+- Dragging on touch devices
+- Mouse not releasing when exiting the window
+- Multiplayer host crashing after beating the game
+- Minor color issues in some videos
+- Command-line output missing on Windows
+- Correct visuals and function of the joining-game-dialog
+- Mouse jumping to the top of the windows when using the menu
+
+### Original Diablo bugs
+- All bugs fixed in Hellfire 1.01
+- Some unique items morphing on a new game
+- Monsters being unable to properly see the location of firewalls
+- Fix more stability issues
+- Gold stacks are correctly topped when in dungeon
+- Gold icons graphics are always correct
+- Arrows graphics not aligning with the move direction
+- Monsters miss calculating charge directions for some angles
+- Randomly being teleported from level 15 to Lazarus chamber
+- Wrong death sound when monster/golem kill another monster
+- Scavengers/Gravediggers taking damage when eating on higher difficulties
+- Not getting XP from monsters hit by golems
+- Monsters with hiding ability fleeing in the wrong direction from golems
+- Monsters going inactive when fighting golems offscreen
+- Gharbad the weak not dropping his item when killed by a golem
+- Lights not always following unique monsters
+- Several bugs in Lazarous quest when re-entering the teleport
+- Lights added on map change not working (Arkaine's Valor etc)
+- Player light radius not being following when a player is knocked back
```

Comment:
```suggestion
- Player light radius not readjusting when a player is knocked back
```
(original seems a bit weird to me)

Reply:
I think "being" was a leftover from another edit :)